draft-ietf-masque-h3-datagram.md

```diff
@@ -623,69 +339,24 @@ recent draft version from the intersection set. This ensures that both
 endpoints negotiate the same draft version.
 
 
-# The Sec-Use-Datagram-Contexts HTTP Header {#context-hdr}
+# The Sec-Capsule-Protocol Header
+
+Definitions of new HTTP Methods or of new HTTP Upgrade Tokens that use the
+Capsule Protocol MAY use the Sec-Capsule-Protocol header to simplify
+intermediary processing.
 
-Endpoints indicate their support for datagram contexts by sending the
-Sec-Use-Datagram-Contexts header with a value of ?1. If the header is missing
-or has a value different from ?1, that indicates that its sender does not wish
-to use datagram contexts. Endpoints that wish to use datagram contexts SHALL
-send the Sec-Use-Datagram-Contexts header with a value of ?1 on requests and
-responses that use the capsule protocol.
+Endpoints indicate that the Capsule Protocol is in use on the data stream by
+sending the Sec-Capsule-Protocol header with a value of ?1. Intermediaries MAY
+use this header to allow processing of HTTP Datagrams for unknown HTTP methods
+or unknown HTTP Upgrade Tokens.
 
-"Sec-Use-Datagram-Contexts" is an Item Structured Header {{!RFC8941}}. Its
-value MUST be a Boolean, its ABNF is:
+"Sec-Capsule-Protocol" is an Item Structured Header {{!RFC8941}}. Its value MUST
+be a Boolean, its ABNF is:
```

Comment:
This bit where you can't define the field as a Boolean directly is weird.  @mnot, why not?

Also, I think that you need to define what to do with multiple values and parameters.

Reply:
I think this is just the standard suggested text for structured fields 🤷

Reply:
Yes, this is standard boilerplate.

I added text about multiple values and parameters.